.github/workflows/ingestion_client.yaml

```diff
@@ -20,7 +46,7 @@ jobs:
         id: get_release_versions
         run: |
           set -u -e -o pipefail
-          version=$(cat ./samples/ingestion/ingestion-client/Setup/ArmTemplateBatch.json | jq -r .variables.Version)
+          version=$(grep '^var Version' ./samples/ingestion/ingestion-client/infra/main.bicep | awk -F "= '" '{print $2}' | sed "s/'//g")
```

Comment:
nit: since you already validated in 'convert_bicep' that the generated matches the committed ARM template, you could keep the jq command and just point to main.json instead. might be a bit more robust to use jq here?